packages/mdc-menu/README.md

```diff
@@ -108,8 +108,9 @@ or a wrapper element that contains the actual visible element to attach to:
 the menu is positioned and displayed correctly.
 
 The menu will check if its parent element has the `mdc-menu-anchor` class set, and if so, it will automatically position
-itself relative to this anchor element. It will open from the top left (top right in RTL) corner of the anchor by
-default, but will choose an appropriate different corner if close to the edge of the screen.
+itself relative to this anchor element. Unless anchor corner is specified, menu will open from the top left
+(top right in RTL) corner of the anchor by default, but will choose an appropriate different corner if close
```

Comment:
Does it actually make sense to swap this in RTL? Sometimes RTL really only makes sense for text based content. Where a menu opens is not as "text-y". (But if theres guidance in our Material Design articles to do this RTL swap, then it makes sense)

Reply:
I think so, in RTL - EVERYTHING is mirrored. I do not think there is any guidance in spec about RTL, I will ask Jefferson.

Reply:
yeah that got fixed